include/odp/api/spec/pool.h

```diff
@@ -176,6 +176,24 @@ int odp_pool_index(odp_pool_t pool);
  */
 int odp_pool_stats(odp_pool_t pool, odp_pool_stats_t *stats);
 
+/**
+ * Get selected statistics for pool handle
+ *
+ * Read the selected counters given in odp_pool_stats_opt_t bit field structure. The selected
+ * counters must have been enabled during pool creation. Values of the unselected counters are
+ * undefined. Depending on the implementation, there may be some delay until performed pool
+ * operations are visible in the statistics.
+ *
+ * @param         pool   Pool handle
+ * @param[out]    stats  Output buffer for counters
+ * @param         opt    Pool counters to read
+ *
+ * @retval  0 on success
+ * @retval <0 on failure
+ */
+int odp_pool_stats_selected(odp_pool_t pool, odp_pool_stats_selected_t *stats,
+			    odp_pool_stats_opt_t *opt);
```

Comment:
odp_pool_stats_opt_t should be "const" since it's read only.

Also, "param opt" documentation could be elaborated a bit: "Bit field for selecting the counters to be read"

Reply:
Changed in V2.